src/software/primitive/move_primitive.h

```diff
@@ -1,32 +1,15 @@
-/**
- * This file includes the definition of the MovePrimitive class and it's member functions
- * and data
- */
-
 #pragma once
 
 #include "software/new_geom/angle.h"
 #include "software/new_geom/point.h"
 #include "software/primitive/primitive.h"
+#include "software/util/make_enum/make_enum.h"
 
-enum AutokickType
-{
-    NONE,
-    AUTOKICK,
-    AUTOCHIP
-};
+MAKE_ENUM(AutochickType, NONE, AUTOKICK, AUTOCHIP);
```

Comment:
I think `AutoChickType` might be a bit better, but I'm cool either way, your call.